app/src/test/java/com/benavalli/arcmovies/features/home/presenter/DetailsActivityPresenterTest.kt

```diff
@@ -0,0 +1,36 @@
+package com.benavalli.arcmovies.features.home.presenter
+
+import android.content.Context
+import com.benavalli.arcmovies.api.repository.HomeRepository
+import com.benavalli.arcmovies.features.home.view.DetailsActivity
+import org.junit.Assert.*
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mock
+import org.mockito.MockitoAnnotations
+import org.mockito.runners.MockitoJUnitRunner
+
+@RunWith(MockitoJUnitRunner::class)
+class DetailsActivityPresenterTest {
```

Comment:
The tests will not run, but you know that, do you intent to finish this? 😄

Reply:
if i can finish i would like.

Reply:
cool